mysql/const.go

```diff
@@ -454,11 +454,23 @@ func FormatSQLModeStr(s string) string {
 	s = strings.ToUpper(strings.TrimRight(s, " "))
 	parts := strings.Split(s, ",")
 	var nonEmptyParts []string
-	for i := 0; i < len(parts); i++ {
-		if len(parts[i]) == 0 {
+	existParts := make(map[string]string)
+	for _, part := range parts {
+		if len(part) == 0 {
 			continue
 		}
-		nonEmptyParts = append(nonEmptyParts, parts[i])
+		if modeParts, ok := combinationSQLMode[part]; ok {
+			for _, modePart := range modeParts {
+				if _, exist := existParts[modePart]; !exist {
+					nonEmptyParts = append(nonEmptyParts, modePart)
+					existParts[modePart] = modePart
+				}
+			}
```

Comment:
Could we continue here?

Reply:
No, the combination mode itself is added too.